sources/can-clab.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Canada's National Park Boundaries dataset", 
+    "edtf:inception": "2021-08-31", 
+    "fullname": "National Framework Canada Lands Administrative Boundaries", 
+    "id": 1746270541, 
+    "key": "", 
+    "license": "https://open.canada.ca/en/open-government-licence-canada", 
+    "license_text": "You are free to: copy, modify, publish, translate, adapt, distribute or otherwise use the Information in any medium, mode or format for any lawful purpose.", 
+    "license_type": "CC BY (equivalent)", 
+    "name": "can-clab", 
+    "prefix": "can-clab", 
+    "url": "https://open.canada.ca/data/en/dataset/9e1507cd-f25c-4c64-995b-6563bf9d65bd", 
+    "usage_concordance": 1, 
+    "usage_geometry": 1, 
+    "usage_property": 0
```

Comment:
There aren't any useful properties to bring in?

Reply:
I didn't add properties at first, but I've updated the admin PR to include top-level `can-clab` properties (names + concordances). I've updated this flag to `1` in a new commit.